frontend/src/shared/components/tasque-task-editing/task-editing.component.ts

```diff
@@ -0,0 +1,324 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { TaskModel } from 'src/core/models/task/task-model';
+import { UserModel } from 'src/core/models/user/user-model';
+import { faCheckToSlot, faXmark, faLink, faPaperclip, faShareNodes, faEllipsisVertical, faFaceSmile, faPen } from '@fortawesome/free-solid-svg-icons';
+import { BaseComponent } from 'src/core/base/base.component';
+import { AngularEditorConfig } from '@kolkov/angular-editor';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ProjectModel } from 'src/core/models/project/project-model';
+import { SprintModel } from 'src/core/models/sprint/sprint-model';
+import { EditorConfig } from 'src/core/settings/angular-editor-setting';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { TaskState } from 'src/core/models/task/task-state';
+import { TaskPriority } from 'src/core/models/task/task-priority';
+
+@Component({
+  selector: 'tasque-task-editing',
+  templateUrl: './task-editing.component.html',
+  styleUrls: ['./task-editing.component.sass']
+})
+export class TaskEditingComponent extends BaseComponent implements OnInit {
+
+  @Input() public task: TaskModel;
+  @Input() public currentUser: UserModel;
+  public taskProject: ProjectModel;
+  public taskUser: UserModel;
+  public taskSprint: SprintModel;
+
+  public editTaskForm: FormGroup = {} as FormGroup;
+
+  public closeIcon = faXmark;
+  public linkIcon = faLink;
+  public checkIcon = faCheckToSlot;
+  public shareIcon = faShareNodes;
+  public paperClipIcon = faPaperclip;
+  public ellipsisIcon = faEllipsisVertical;
+  public faceSmileIcon = faFaceSmile;
+  public editIcon = faPen;
```

Comment:
why not to use fa-icons for all of them ?

Reply:
Because our tasque-button already has fa-icon using.